app/views/course/assessment/answer/programming/_annotation_post.html.slim

```diff
@@ -1,4 +1,9 @@
-= div_for(annotation_post) do
+= div_for(annotation_post, 'data-post-id' => annotation_post.id) do
+  div.toolbar.pull-right
+    - if can?(:destroy, annotation_post)
+      = link_to('#', class: ['delete']) do
```

Comment:
@lowjoel we have two versions of `link_to` in our codebase now (with brackets and without brackets), think it need to standardised.